Makefile

```diff
@@ -352,6 +352,7 @@ release: clean-release  ## Builds and push container images using the latest git
 	$(MAKE) set-manifest-pull-policy PULL_POLICY=IfNotPresent
 	$(MAKE) release-manifests
 	$(MAKE) release-templates
+	cp metadata.yaml $(RELEASE_DIR)/metadata.yaml
```

Comment:
let's break this out in a separate make target like release-templates

Reply:
ok

Reply:
done @CecileRobertMichon thanks for the review